src/ui/atoms/ContentToggle.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { cleanup, render, RenderResult } from '@testing-library/react';
+import ContentToggle from './ContentToggle';
+import SelectField from './SelectField';
+
+describe('ContentToggle', (): void => {
+    afterEach(cleanup);
+
+    it('should render correctly', (): void => {
+        expect(
+            (): RenderResult =>
+                render(
+                    <ContentToggle id="" collapsedText="" openText="">
+                        This is some text
```

Comment:
Because `children` is an optional prop and this test case should be testing the render of minimal props passed to the component, we can remove the text content here and make this a self closing tag.